torch/csrc/jit/passes/quantization.cpp

```diff
@@ -83,6 +83,7 @@ bool nodeQuantizable(Node* n) {
       "conv2d",
       "linear",
       "relu",
+      "_convolution",
```

Comment:
How about we call `replaceConvolutionWithConv2d` in the very beginning, before we even inserted observers? In this case we won't need to extend the special handling of `conv2d` to `_convolution`. For instance, you've already forgotten to update one function: `isBiasOfConvOrLinear`.